frontend/src/components/Profile.jsx

```
@@ -0,0 +1,46 @@
+            <p className="text-gray-600">Name:</p>
+            <p className="text-gray-800">{name || "Not provided"}</p>
+          </div>
+          <div>
+            <p className="text-gray-600">Email:</p>
+            <p className="text-gray-800">{email || "Not provided"}</p>
+          </div>
+        </div>
+        <button
+          onClick={onClose}
+          className="mt-6 w-full bg-gray-700 text-white py-2 rounded hover:bg-gray-800 transition-colors"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)